File: app/src/main/java/com/cn29/aac/ui/location/LocationActivity.java
package com.cn29.aac.ui.location;

import android.os.Bundle;

import com.cn29.aac.R;
import com.cn29.aac.ui.base.BaseAppCompatActivity;
import com.cn29.aac.ui.location.vm.LastLocationViewModel;

import javax.inject.Inject;

import androidx.appcompat.widget.Toolbar;

public class LocationActivity extends BaseAppCompatActivity {

  @Inject
  LastLocationViewModel lastLocationViewModel;

  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_location);
    Toolbar toolbar = findViewById(R.id.toolbar);
    setSupportActionBar(toolbar);
  }


}

File: app/src/main/java/com/cn29/aac/ui/location/LocationActivity.kt
package com.cn29.aac.ui.location

import android.os.Bundle
import androidx.appcompat.widget.Toolbar
import com.cn29.aac.R
import com.cn29.aac.ui.base.BaseAppCompatActivity
import com.cn29.aac.ui.location.vm.LastLocationViewModel
import javax.inject.Inject

class LocationActivity : BaseAppCompatActivity() {
    @JvmField
    @Inject
    var lastLocationViewModel: LastLocationViewModel? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_location)
        val toolbar = findViewById<Toolbar>(
                R.id.toolbar)
        setSupportActionBar(toolbar)
    }
}